{"timestamp":"2025-09-07T14:08:14.876147","level":"info","event":"DAG bundles loaded: dags-folder, example_dags","logger":"airflow.dag_processing.bundles.manager.DagBundlesManager"}
{"timestamp":"2025-09-07T14:08:14.876653","level":"info","event":"Filling up the DagBag from /opt/airflow/dags/pipeline.py","logger":"airflow.models.dagbag.DagBag"}
{"timestamp":"2025-09-07T14:08:15.200995Z","level":"info","event":"Task instance is in running state","chan":"stdout","logger":"task"}
{"timestamp":"2025-09-07T14:08:15.201247Z","level":"info","event":" Previous state of the Task instance: TaskInstanceState.QUEUED","chan":"stdout","logger":"task"}
{"timestamp":"2025-09-07T14:08:15.201416Z","level":"info","event":"Current task name:import_data","chan":"stdout","logger":"task"}
{"timestamp":"2025-09-07T14:08:15.201568Z","level":"info","event":"Dag name:extract_and_store_retail_data","chan":"stdout","logger":"task"}
{"timestamp":"2025-09-07T14:08:15.202628","level":"error","event":"Task failed with exception","logger":"task","error_detail":[{"exc_type":"TypeError","exc_value":"import_data_and_transform() takes 0 positional arguments but 1 was given","exc_notes":[],"syntax_error":null,"is_cause":false,"frames":[{"filename":"/home/airflow/.local/lib/python3.12/site-packages/airflow/sdk/execution_time/task_runner.py","lineno":920,"name":"run"},{"filename":"/home/airflow/.local/lib/python3.12/site-packages/airflow/sdk/execution_time/task_runner.py","lineno":1215,"name":"_execute_task"},{"filename":"/home/airflow/.local/lib/python3.12/site-packages/airflow/sdk/bases/operator.py","lineno":397,"name":"wrapper"},{"filename":"/home/airflow/.local/lib/python3.12/site-packages/airflow/providers/standard/operators/python.py","lineno":216,"name":"execute"},{"filename":"/home/airflow/.local/lib/python3.12/site-packages/airflow/providers/standard/operators/python.py","lineno":239,"name":"execute_callable"},{"filename":"/home/airflow/.local/lib/python3.12/site-packages/airflow/sdk/execution_time/callback_runner.py","lineno":81,"name":"run"}],"is_group":false,"exceptions":[]}]}
{"timestamp":"2025-09-07T14:08:15.242709Z","level":"info","event":"Task instance in failure state","chan":"stdout","logger":"task"}
{"timestamp":"2025-09-07T14:08:15.243057Z","level":"info","event":"Task start","chan":"stdout","logger":"task"}
{"timestamp":"2025-09-07T14:08:15.243290Z","level":"info","event":"Task:<Task(PythonOperator): import_data>","chan":"stdout","logger":"task"}
{"timestamp":"2025-09-07T14:08:15.243501Z","level":"info","event":"Failure caused by import_data_and_transform() takes 0 positional arguments but 1 was given","chan":"stdout","logger":"task"}
